pages/common/deno.md

```diff
@@ -0,0 +1,25 @@
+# deno
+
+> A secure runtime for JavaScript and TypeScript
+> More information: <https://deno.land/>.
+
+- Run a JavaScript or TypeScript file:
+
+`deno file.ts`
+
+- Start a REPL (interactive shell):
+
+`deno`
+
+- Run a file with access to the network:
```

Comment:
```suggestion
- Run a file with the access to the network:
```

Reply:
I don't think this suggestion sounds quite right. What do you think of: "Run a file with network access"?

Reply:
This is correct as is, maybe a clearer version could be:

    - Run a file with network access enabled:

which is indeed what the command does.